src/main/java/org/mule/tools/apikit/model/ScaffoldingAccessories.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.tools.apikit.model;
+
+public class ScaffoldingAccessories {
+
+  private final boolean showConsole;
+  private String externalCommonFile;
+  private String apiId;
+  private Properties properties;
+
+  public ScaffoldingAccessories() {
```

Comment:
remove this, add a method for each field of this class in ScaffoldingConfiguration and construct ScaffoldingAccessories in ScaffoldingConfiguration.Builder.build()